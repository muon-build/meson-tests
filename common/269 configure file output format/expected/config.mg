#ifndef CONFIG_MAGNESIUM_H
#define CONFIG_MAGNESIUM_H
#define str "hello world!"
#define unquoted float
#define int 42
#define bool
#undef false
#endif
