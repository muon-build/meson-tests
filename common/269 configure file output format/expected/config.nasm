%define str "hello world!"
%define unquoted float
%define int 42
%define bool
%undef false
