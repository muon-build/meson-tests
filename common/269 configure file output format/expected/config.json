{"str":"\"hello world!\"","unquoted":"float","int":42,"bool":true,"false":false}
